java/tables/src/main/java/sleeper/table/job/TableInitialiser.java

```diff
@@ -76,13 +77,16 @@ private List<Object> getSplitPoints(TableProperties tableProperties, String conf
             return null;
         }
         List<Object> splitPoints = new ArrayList<>();
-        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(s3.getObject(configBucket, splitsFile).getObjectContent()));
-        boolean stringsBase64Encoded = Boolean.parseBoolean(tableProperties.get(SPLIT_POINTS_BASE64_ENCODED));
-        PrimitiveType keyType = tableProperties.getSchema().getRowKeyTypes().get(0);
-        bufferedReader.lines()
-                .map(l -> createSplitPoint(l, stringsBase64Encoded, keyType))
-                .forEach(splitPoints::add);
-
+        try (InputStreamReader inputStreamReader = new InputStreamReader(s3.getObject(configBucket, splitsFile).getObjectContent(), StandardCharsets.UTF_8);
+             BufferedReader bufferedReader = new BufferedReader(inputStreamReader)) {
+            boolean stringsBase64Encoded = Boolean.parseBoolean(tableProperties.get(SPLIT_POINTS_BASE64_ENCODED));
+            PrimitiveType keyType = tableProperties.getSchema().getRowKeyTypes().get(0);
+            bufferedReader.lines()
+                    .map(l -> createSplitPoint(l, stringsBase64Encoded, keyType))
+                    .forEach(splitPoints::add);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

Comment:
Is this just for UnsupportedEncodingException? Maybe that should be declared?

Reply:
Not just UnsupportedEncodingException, but OS_OPEN_STREAM (stream may fail to close).